tini/src/seccomp_fd_notify.c

```diff
@@ -31,6 +31,7 @@
 #endif
 
 #define WAIT_KILLABLE_SECCOMP_KERNEL_VERSION KERNEL_VERSION(5, 15, 35)
+#define WAIT_KILLABLE_SECCOMP_EXTRA_KERNEL_VERSION "netflix-g54efd87a8576"
```

Comment:
This is not super sustainable. As soon as we do an upgrade, which will still have all the seccomp goodies, won't it break?

Don't you really need a blacklist of 5.15.35 kernels that don't support this feature?

Reply:
I agree. I don't think these kinds of version checks are really that reliable anyway, since people can backport, etc. I'd do it like the kernel test suite or libseccomp does it, and just check for EINVAL from the actual syscall: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/tools/testing/selftests/seccomp/seccomp_bpf.c#n2377

Reply:
Yeah I was checking if the kernel version was greater than 5.15.35 or equal and extra version was netflix-g54efd87a8576, but yeah as @tych0 said someone could backport and we can have a new version of 5.15.35 and this check will break. I'll add a check to see what flags are supported as @tych0 suggested.

Reply:
@tych0 @solarkennedy do the new changes look ok?